### ISSUE TITLE
bad performance than expected

### ISSUE THREAD

**Issue by JXL884** [AnonymousAKES/Differentially-Private-Fine-tuning-of-Language-Models#2]
Hi there,

As I trying to replicate your results, I followed the instructions and ran the bert-lora with full-precision using your command.

python run_exp.py --gpu_id 0 --task SST-2 --k 16 --eps 8 --delta 1e-5 --clip 10. --accountant moments --batch_size 2000 --lr 1e-3 --epoch 50  --sess lora_bertx_debug --to_console --fp32

But I only got 70% accuracy as best performance. I'm wondering if you came across same issue or similar complains before, thank you so much

**Comment by dayu11** [comment]
Thanks for the question! Based on the description, it seems that the pre-trained checkpoint is not correctly downloaded or placed at a wrong location. Please check the last paragraph of [this page](https://github.com/huseyinatahaninan/Differentially-Private-Fine-tuning-of-Language-Models/tree/main/Language-Understanding-RoBERTa) for instructions.

Please let me know if this does not help.
